packages/minizip.rb

```diff
@@ -1,40 +1,34 @@
 require 'package'
+require_relative 'zlibpkg'
```

Comment:
Use `Package.load_package()`? (a wrapper for loading package templates)

Reply:
`Package.load_package('zlibpkg')` doesn't work instead of `require_relative 'zlibpkg'`...